### ISSUE TITLE
Unable to navigate to the suggestion in the search field when the narrator is on.

### ISSUE THREAD

**Issue by msftedad** [TypeStrong/typedoc#2563]
## Expected Behavior
unable to navigate to the suggestion in the search field when narrator is on. screen reader is failed to announce the search suggestions.

## Actual Behavior
narrator should announce the search suggestion and focus should move to the search suggestions.

## Steps to reproduce the bug

1.Open https://typedoc.org/api/in edge browser.
2.Navigate to search field which is at top right and invoke it.
3.enter some text and search suggestions will be displayed.
4.now try to navigate to search suggestions when the screen reader is on.

[d3856682-230a-47d3-bac7-aa12474629ea.webm](https://github.com/TypeStrong/typedoc/assets/93735775/5290aa32-5a09-49f5-a748-c4c56d6c420b)

## Environment

-   Typedoc version: v0.25.13
-   TypeScript version:
-   Node.js version:
-   OS: 23H2 ((OS Build 22631.3447)

**Comment by Gerrit0** [comment]
Narrator is... infuriating... but I believe I have bludgeoned it into submission for now...

**Comment by Gerrit0** [comment]
Fixed with TypeDoc 0.26, which is releasing 2024/06/21
